composer.json

```diff
@@ -57,13 +57,13 @@
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/mongodb-odm": "^2.3",
         "doctrine/orm": "^2.10.2",
-        "friendsofphp/php-cs-fixer": "^3.4.0 <3.10",
+        "friendsofphp/php-cs-fixer": "^3.4.0",
```

Comment:
The `groups` option for the `phpdoc_separation` rule was introduced in [version 3.14.0](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/releases/tag/v3.14.0).

See PHP-CS-Fixer/PHP-CS-Fixer#6668.

Reply:
And https://packagist.org/packages/friendsofphp/php-cs-fixer#v3.14.0 is `php: ^7.4 || ^8.0`
In that case, https://github.com/doctrine-extensions/DoctrineExtensions/pull/2636#issuecomment-1685240982 could be merged upfront?

Reply:
Yes, we should bump the PHP version before.